src/main/java/com/epam/spring/time_tracking/dto/activity/ActivityForUserDto.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.spring.time_tracking.dto.activity;
+
+import com.epam.spring.time_tracking.dto.category.CategoryDto;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Data
+public class ActivityForUserDto {
+
+    private int id;
+    private String name;
+    private List<CategoryDto> categories;
+    private String description;
+    private String image;
+    private int peopleCount;
```

Comment:
I can see the duplicated fields like, id, name, peopleCount and etc. in all Activity dto classes, maybe try to use some inheritance to avoid such duplication?